src/comments.ts

```diff
@@ -0,0 +1,65 @@
+class NTRComment {
+    text: string;
+    nodeID: string;
+    startOffset: number;
+    endOffset: number;
+
+    constructor(text: string, nodeID: string, startOffset: number, endOffset: number) {
+        this.text = text;
+        this.nodeID = nodeID;
+        this.startOffset = startOffset;
+        this.endOffset = endOffset;
+    }
+}
+
+/// Temporary function for interpreting the demo comments
+const interpComments = (comments: string): NTRComment[] => {
```

Comment:
This is a somewhat messy function that could be greatly simplified by just storing comments as a json file, but since it's temporary I'll let it slide.